docs/ros/ros/installation/remote_pc.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: Remote PC setup
```

Comment:
We have been using the term "Computer" throughout the rest of the documentation rather than "PC". Can you change all "PC" to "Computer"?